src/oca_github_bot/tasks/main_branch_bot.py

```diff
@@ -14,16 +14,16 @@
 
 
 @switchable("gen_addons_table")
-def _gen_addons_table(org, repo, branch):
+def _gen_addons_table(org, repo, branch, cwd):
     _logger.info("oca-gen-addons-table in %s/%s@%s", org, repo, branch)
-    gen_addons_table_cmd = ["oca-gen-addons-table", "--commit"]
+    gen_addons_table_cmd = ["oca-gen-addons-table", "--commit", "--addons-dir", cwd]
     subprocess.check_output(
         gen_addons_table_cmd, universal_newlines=True, stderr=subprocess.STDOUT
     )
```

Comment:
@SimoRubi an easy fix is to pass `cwd` to `subprocess.check_output`, no need to change `oca-gen-addons-table` for that.

Reply:
Thanks, I've implemented the fix.
I'll leave the issue in maintainer tools so that `oca-gen-addons-table` can be more aligned with the other methods, anyway it will be discussed in the issue itself.